go.mod

```diff
@@ -19,112 +19,112 @@ replace cuelang.org/go => github.com/sdboyer/cue v0.5.0-beta.2.0.20230419165817-
+	github.com/robfig/cron/v3 v3.0.1 // @grafana/backend-platform
+	github.com/russellhaering/goxmldsig v1.4.0 // @grafana/backend-platform
+	github.com/stretchr/testify v1.8.4 // @grafana/alerting-squad-backend
+	github.com/teris-io/shortid v0.0.0-20171029131806-771a37caa5cf // @grafana/backend-platform
+	github.com/ua-parser/uap-go v0.0.0-20211112212520-00c877edfe0f // @grafana/backend-platform
 	github.com/uber/jaeger-client-go v2.29.1+incompatible // indirect
-	github.com/urfave/cli/v2 v2.25.0
-	github.com/vectordotdev/go-datemath v0.1.1-0.20220323213446-f3954d0b18ae
-	github.com/yalue/merged_fs v1.2.2
-	github.com/yudai/gojsondiff v1.0.0
-	go.opentelemetry.io/collector v0.31.0
-	go.opentelemetry.io/collector/model v0.31.0
-	go.opentelemetry.io/contrib/instrumentation/net/http/httptrace/otelhttptrace v0.37.0
-	go.opentelemetry.io/otel v1.14.0
-	go.opentelemetry.io/otel/exporters/jaeger v1.0.0
-	go.opentelemetry.io/otel/sdk v1.14.0
-	go.opentelemetry.io/otel/trace v1.14.0
-	golang.org/x/crypto v0.11.0
-	golang.org/x/exp v0.0.0-20230321023759-10a507213a29
-	golang.org/x/net v0.10.0
-	golang.org/x/oauth2 v0.7.0
-	golang.org/x/sync v0.3.0
-	golang.org/x/time v0.3.0
-	golang.org/x/tools v0.7.0
-	gonum.org/v1/gonum v0.11.0
-	google.golang.org/api v0.111.0
-	google.golang.org/grpc v1.55.0
-	google.golang.org/protobuf v1.30.0
+	github.com/urfave/cli/v2 v2.25.0 // @grafana/backend-platform
+	github.com/vectordotdev/go-datemath v0.1.1-0.20220323213446-f3954d0b18ae // @grafana/backend-platform
+	github.com/yalue/merged_fs v1.2.2 // @grafana/grafana-as-code
+	github.com/yudai/gojsondiff v1.0.0 // @grafana/backend-platform
+	go.opentelemetry.io/collector v0.31.0 // @grafana/backend-platform
+	go.opentelemetry.io/collector/model v0.31.0 // @grafana/backend-platform
+	go.opentelemetry.io/contrib/instrumentation/net/http/httptrace/otelhttptrace v0.37.0 // @grafana/grafana-operator-experience-squad
+	go.opentelemetry.io/otel v1.14.0 // @grafana/alerting-squad-backend
+	go.opentelemetry.io/otel/exporters/jaeger v1.0.0 // @grafana/backend-platform
+	go.opentelemetry.io/otel/sdk v1.14.0 // @grafana/backend-platform
+	go.opentelemetry.io/otel/trace v1.14.0 // @grafana/backend-platform
+	golang.org/x/crypto v0.11.0 // @grafana/backend-platform
+	golang.org/x/exp v0.0.0-20230321023759-10a507213a29 // @grafana/alerting-squad-backend
+	golang.org/x/net v0.10.0 // @grafana/grafana-bi-squad
+	golang.org/x/oauth2 v0.7.0 // @grafana/backend-platform
+	golang.org/x/sync v0.3.0 // @grafana/alerting-squad-backend
+	golang.org/x/time v0.3.0 // @grafana/backend-platform
+	golang.org/x/tools v0.7.0 // @grafana/grafana-as-code
+	gonum.org/v1/gonum v0.11.0 // @grafana/observability-metrics
+	google.golang.org/api v0.111.0 // @grafana/backend-platform
+	google.golang.org/grpc v1.55.0 // @grafana/backend-platform
+	google.golang.org/protobuf v1.30.0 // @grafana/grafana-app-platform-squad
```

Comment:
```suggestion
	google.golang.org/protobuf v1.30.0 // @grafana/plugins-platform-backend
```

Reply:
likely also grpc?

Reply:
@mildwonkey suggested this could also be ours so i will assign it to BEP for now